### PR TITLE
Update syn and bump version

### PR DESCRIPTION
**Pull request by Eijebong**
This change is [Reviewable](https://reviewable.io/reviews/servo/rust-cssparser/229)

### CODE REVIEW

File: build/match_byte.rs [hunk 1]
```diff
@@ -27,20 +29,18 @@ pub fn expand(from: &Path, to: &Path) {
 
 struct MatchByte {
     expr: syn::Expr,
-    arms: Vec<syn::Arm>,
+    _comma: Token![,],
+    arms: Punctuated<syn::Arm, Option<Token![,]>>,
```
Comment (dtolnay) [inline build/match_byte.rs]: The `Arm` parser will always eat a trailing comma if there is one. https://github.com/dtolnay/syn/blob/0.15.12/src/expr.rs#L2777-L2781. The comma will go in https://docs.rs/syn/0.15/syn/struct.Arm.html#structfield.comma.

I would recommend parsing these as Vec\<Arm\> as seen in https://github.com/dtolnay/syn/blob/0.15.12/src/expr.rs#L1898-L1901.

File: build/match_byte.rs [hunk 2]
```diff
@@ -27,20 +29,18 @@ pub fn expand(from: &Path, to: &Path) {
 
 struct MatchByte {
     expr: syn::Expr,
-    arms: Vec<syn::Arm>,
+    _comma: Token![,],
```
Comment (dtolnay) [inline build/match_byte.rs]: I would leave this out of the struct if it won't be used.

**Comment by SimonSapin** [comment]
https://crates.io/crates/cssparser/0.25.1

**Pull request merged**